{"name": "Orders list get failed for non auth user test", "status": "passed", "start": 1727809279410, "stop": 1727809279543, "uuid": "f1cf83bb-6cb9-4e3b-be7e-953956ce952c", "historyId": "87f757644590ec200a5414bbff240c4f", "testCaseId": "87f757644590ec200a5414bbff240c4f", "fullName": "tests.test_get_orders.TestGetOrders#test_get_orders_non_auth_user", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_get_orders"}, {"name": "subSuite", "value": "TestGetOrders"}, {"name": "host", "value": "WIN-B95O9QQ3RDO"}, {"name": "thread", "value": "23340-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_get_orders"}]}